casatables/src/glue.cc

```diff
@@ -131,8 +133,130 @@ extern "C" {
         return -1;
     }
 
+    // TableDesc
+
+    GlueTableDesc *
+    tabledesc_create(
+        // TODO: figure out exactly what this is.
+        // Apparently this is actually the name of the table?
+        const StringBridge &type
+    )
+    {
+        // TODO: expose this?
+        GlueTableDesc::TDOption tdOption = GlueTableDesc::TDOption::New;
+        return new GlueTableDesc(bridge_string(type), tdOption);
+    }
+
+    GlueTableDesc *
+    tabledesc_add_scalar_column(
+        GlueTableDesc &tableDesc,
+        GlueDataType dataType,
+        const StringBridge &columnName,
+        ExcInfo &exc
+    )
+    {
+        try {
+
+            switch (dataType) {
```

Comment:
Stylistically, I like to blank lines a lot, but I don't put them after lines ending with open braces.